doc/man3/X509_ALGOR_dup.pod

```diff
@@ -36,12 +37,15 @@ values for the message digest B<md>.
 X509_ALGOR_cmp() compares B<a> and B<b> and returns 0 if they have identical
 encodings and nonzero otherwise.
 
+X509_ALGOR_copy() copies the source values into the dest structs; making
+a duplicate of each (and free any thing pointed to from within *dest).
+
 =head1 RETURN VALUES
 
 X509_ALGOR_dup() returns a valid B<X509_ALGOR> structure or NULL if an error
 occurred.
 
-X509_ALGOR_set0() returns 1 on success or 0 on error.
+X509_ALGOR_set0() and _X509_ALGOR_copy() return 1 on success or 0 on error.
```

Comment:
Typo - superfluous underscore before the X509_ALGOR_copy. Also there is missing a HISTORY note that it was added in 1.1.1e.